recipe/meta.yaml

```diff
@@ -125,8 +119,6 @@ outputs:
         - click >=8.1.7
         - coloredlogs >=15.0.1
         - colour-science >=0.4.4
-        # conda is required for runtime inspection of installed software
-        - conda >=23.11.0
```

Comment:
Why was this removed?

Reply:
Because we don't need it any more. We no longer use the python conda library to get the `conda list` output for versioning.

Reply:
See e.g. https://github.com/conda/conda/issues/13115